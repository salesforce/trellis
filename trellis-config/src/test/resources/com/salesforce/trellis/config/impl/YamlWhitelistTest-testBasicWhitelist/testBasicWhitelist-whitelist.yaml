#
# Generated file.  DO NOT EDIT
# See https://sfdc.co/trellisFaq for more information.
#
---
rules:


- action: WARN
  from:
  - sfdc.core:no-longer-violating-module
  to:
  - sfdc.core:newly-acceptable-module
  reason: This should be removed from the generated whitelist because it no longer depends on junit.

- action: WARN
  from:
  - sfdc.core:unprocessed-module
  to:
  - junit:junit
  reason: This should be retained in the updated whitelist because the module was not processed
