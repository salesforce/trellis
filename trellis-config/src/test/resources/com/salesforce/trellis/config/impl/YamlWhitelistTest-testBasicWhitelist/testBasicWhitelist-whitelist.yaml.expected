#
# This is the header comment for testBasicWhitelist.
#
rules:

- action: WARN
  from:
  - sfdc.core:another-whitelisted-model
  - sfdc.core:newly-whitelisted-model
  - sfdc.core:platform-encryption
  to:
  - junit:junit
  reason: Production code shouldn't depend on test frameworks.
  scope: compile

- action: WARN
  from:
  - sfdc.core:unprocessed-module
  to:
  - junit:junit
  reason: This should be retained in the updated whitelist because the module was not processed